examples/algorithms/lra_csv_phyfun.py

```diff
@@ -0,0 +1,30 @@
+import phylanx
+et = phylanx.execution_tree
+from phylanx.util import *
+
+@phyfun
+def lra(x, y, alpha, iterations, enable_output):
+        weights=constant(0.0, shape(x, 1))
```

Comment:
This file uses 8 spaces for indentation. Could we please agree on an indentation width for all Python files?